module/zfs/vdev_raidz_math.c

```diff
@@ -625,49 +628,43 @@ vdev_raidz_impl_set(const char *val)
 
 	if (err == 0) {
 		if (raidz_math_initialized)
-			atomic_swap_32(&zfs_vdev_raidz_impl, impl);
+			atomic_swap_32(&zfs_vdev_raidz_impl_setting, impl);
 		else
 			atomic_swap_32(&user_sel_impl, impl);
+#if defined(__linux__)
+		zfs_vdev_raidz_impl = raidz_supp_impl[i]->name;
+#endif
```

Comment:
I don't see anything reading this variable.  It might be needed to exist just for sake of the macro, but with explicitly defined get and set functions it seems nothing accesses it.

Reply:
Yes, that's correct.  The macro requires it.  Without it, the build fails.

Reply:
But you don't need to initialize or assign it.